### DISCUSSION TITLE
如何停止发音的音频文件播放？

### DISCUSSION THREAD

**Discussion by bennyyip** [xiaoyifang/goldendict-ng#1078] [Q&A]
复现步骤：
1. 将 Wikipedia 加入词典
2. 访问 goldendict://trumpet

GoldenDict 会开始播放一段小号演奏的音频，如何让它停止播放？

**Comment by xiaoyifang** [comment]
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/7c198c48-2bf3-4a80-a702-cb32d9e227a9)
不要勾选

**Comment by bennyyip** [comment]
但是我想要自動播放，只是遇到這種長長的音頻想手動停止

**Comment by xiaoyifang** [comment]
目前没有办法，后续看情况，加个快捷键用来停止播放的功能

**Comment by xiaoyifang** [comment]
https://github.com/xiaoyifang/goldendict-ng/pull/1081

**Comment by xiaoyifang** [comment]
最新版，可以按   Ctrl+Shift+S  停止当前的音频播放。

**Comment by bennyyip** [comment]
牛的
